addons/external-dns/external-dns.yaml

```diff
@@ -24,9 +24,12 @@ spec:
     - name: none
       enabled: false
   chartReference:
-    repo: https://charts.bitnami.com/bitnami
+    repo: https://mesosphere.github.io/charts/stable
     chart: external-dns
-    version: 2.20.4
+    version: 2.20.5
     values: |
       rbac:
         create: true
+      service:
+        labels:
+          servicemonitor.kubeaddons.mesosphere.io/path: "metrics"
```

Comment:
you don't need to add a name like for other fixes? eg [like this](https://github.com/mesosphere/charts/pull/851/files#diff-e1d2b0bcd42907e6b6290c1d289d57454e245189bb550886e6746ea7a38f4944R9)

Reply:
for external-dns, we can reuse an existing servicemonitor that just selects on that one label (https://github.com/mesosphere/kubernetes-base-addons/blob/master/addons/prometheus/prometheus.yaml#L83-L86)